Author: Pino Toscano <pino@debian.org>
Description: Fix the .desktop files
 Remove the Encoding keys, deprecated.
Last-Update: 2011-09-06
Forwarded: no

--- a/gadgets/designer/ggl-designer.desktop.in
+++ b/gadgets/designer/ggl-designer.desktop.in
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=GADGET_DESIGNER_NAME
 _Comment=GADGET_DESIGNER_COMMENT
 TryExec=@bindir@/ggl-gtk
--- a/hosts/gtk/ggl-gtk.desktop.in
+++ b/hosts/gtk/ggl-gtk.desktop.in
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=HOST_GTK_DESKTOP_NAME
 _Comment=HOST_GTK_DESKTOP_COMMENT
 TryExec=@bindir@/ggl-gtk
--- a/hosts/qt/ggl-qt.desktop.in
+++ b/hosts/qt/ggl-qt.desktop.in
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=HOST_QT_DESKTOP_NAME
 _Comment=HOST_QT_DESKTOP_COMMENT
 TryExec=@bindir@/ggl-qt
